### ISSUE TITLE
A beginner guide to SoftMax Regression using TensorFlow

### ISSUE THREAD

**Issue by Wanyoike-shift** [section-engineering-education/engineering-education#6879]
#### Proposed title of article
A beginner guide to SoftMax Regression using TensorFlow

#### Proposed article introduction
When compared to normal classification/regression Machine Learning problems like as house prices or Titanic survivors, this one is far more difficult to answer. Using the pixels that represent the numbers in this problem, we can figure out how to solve the riddle. There is a data collection that includes pixels at different positions on the grid of numbers that we are provided with. This is referred described as a "Deep Learning" problem since it involves a more advanced use of machine learning. When classifying the data, we will utilize a Softmax Regression model that has been SGD-optimized to aid us.

#### Key takeaways 
- Understanding what is Softmax Regression
- An easy-to-understand guide to linear regression
- Implement Softmax Regression on the MNIST handwritten digit dataset using TensorFlow library

#### Article quality
This tutorial is meant for people who are new to machine learning as well as those who are new to the Kaggle community. We will use a straightforward approach known as SoftMax regression to tackle the classification issue for handwritten digits in the MNIST dataset.

#### References
N/A

**Comment by WanjaMIKE** [comment]
Seems like a helpful topic. Let's ensure that it adds value to the community. Thanks

**Comment by WanjaMIKE** [comment]
Closed #6879 via #7477
